src/config.rs

```diff
@@ -2329,6 +2329,8 @@ pub struct CdcConfig {
     #[doc(hidden)]
     #[serde(skip_serializing)]
     pub old_value_cache_size: usize,
+    #[online_config(skip)]
+    pub api_version: u8,
```

Comment:
Pass api_version to cdc::Endpoint on initialization [here](https://github.com/tikv/tikv/blob/b60271983288caf7d978f111cd1878b508861327/components/server/src/server.rs#L919).

It is not good to add a item here. CdcConfig is the [cdc] item in config file. As api-version is already configured in [storage], it should be not configured again in a second place.

comment from @pingyu